src/wrapper/communicator.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         components: List[Component] = None,
         max_tick: int = 86_400,
         sumo_port: int = None,
-        sumo_configuration: str = "sumo-config/example.scenario.sumocfg",
+        sumo_configuration: str = "data/sumo/example/sumo-config/example.scenario.sumocfg",
```

Comment:
I recommend you to use path.join for this 😇 It differentiates between OS if you have to use `/` or `\`.